person.rb

```diff
@@ -2,13 +2,14 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
 
   def initialize(id, age, name = 'Unknow')
     super()
     @id = id
     @name = name
     @age = age
+    @rental = []
     @parent_permission = true
   end
```

Comment:
- [ ] Dear @hienphan0111 `parent_permission = true` should be set as a parameter, that way if the user provides a value while initializing a person the default `true` would be replaced by the new value. As the way you set it now, We are saying it is always true and that is not the expected behavior.